scripts/qtsupport.py

```diff
@@ -507,20 +508,115 @@ def filter(self, p, columns):
             item.setHidden(all([unicode(item.text(column)).lower().find(p) == -1
                                 for column in columns]))
 
-""" TODO implement this option
-class SchStaticPage(QWizardPage):
-    def __init__(self, parent):
-        super(SchStaticPage, self).__init__(parent)
-        self.setTitle("Manually create a schedule entry")
+# TODO implement this option
+#class SchStaticPage(QWizardPage):
+#    def __init__(self, parent):
+#        super(SchStaticPage, self).__init__(parent)
+#        self.setTitle("Manually create a schedule entry")
+#        layout = QGridLayout()
+#        wdgts = getSettingsWidgets()
+#        for i, x in enumerate(wdgts):
+#            layout.addWidget(x[0], i + 1, 0)
+#            layout.addWidget(x[1], i + 1, 1, 1, 2)
+#        wdgts[0][1].editingFinished.connect(
+#                    lambda: checkAddress(self, wdgts[0][1].text()))
+#        self.setLayout(layout)
+
+
+class BitcoinAmountBTCValidator(QDoubleValidator):
+
+    def __init__(self):
+        super().__init__(0.00000000, 20999999.9769, 8)
```

Comment:
Given my report here: https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/530#issuecomment-630755454 this looks like something weird is going on, you set the accuracy correctly ...

Reply:
I override `validate()` method, that's why it didn't work. :)